AsyncDisplayKit/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -14,6 +14,27 @@
 
 #pragma mark - UIAccessibilityElement
 
+// Sort accessiblity elements by x and y origin. First sort the elements by y and than by x value.
+static NSArray *SortAccessibilityElements(NSArray *accessibleElements) {
```

Comment:
This is great!  Should we use NSMutableArray?  Less functional, but probably more efficient.  If we did, we'd probably make this function return just the comparator block, and then call the sort method on the mutable array directly.